app/lib/system/error_reporting.rb

```diff
@@ -16,7 +16,6 @@ def report_error(exception, rack_env: nil, logger: Rails.logger, **parameters)
 
       logger.error('Exception') { {exception: {class: exception.class, message: (exception.try(:message) || exception.to_s), backtrace: (exception.try(:backtrace) || [])[0..3]}, parameters: parameters} }
 
-      ::Airbrake.notify_or_ignore(exception, options) if defined?(Airbrake)
```

Comment:
This implies that we don't need to declare the `options` of the first line of the method anymore :smile: